skimage/filters/lpi_filter.py

```diff
@@ -195,7 +200,7 @@ def inverse(data, impulse_response=None, filter_params={}, max_gain=2,
         filt = predefined_filter
 
     F, G = filt._prepare(data)
-    _min_limit(F)
+    _min_limit(F, val=np.finfo(float).eps)
```

Comment:
```suggestion
    _min_limit(F, val=eps)
```
`eps` is already defined as a global variable. But I am not against your writing, in this case I suggest removing the global declaration of `eps`, just for consistency :wink: